modules/database/src/ioc/db/dbAccessDefs.h

```diff
@@ -106,7 +106,8 @@ epicsShareExtern int dbAccessDebugPUTF;
 	epicsUInt16	status;		/* alarm status */\
 	epicsUInt16	severity;	/* alarm severity*/\
 	epicsUInt16	acks;		/* alarm ack severity*/\
-	epicsUInt16	ackt;		/* Acknowledge transient alarms?*/
+    epicsUInt16	ackt;		/* Acknowledge transient alarms?*/\
+    char amsg[48];
```

Comment:
Curious: Why is this 48? Making it larger than MAX_STRING_SIZE=40 seems likely to be occasionally annoying.

Reply:
I recall having the alignment wrong at one point (must be a multiple of 8) and rounded up.  Reducing to 40 seems reasonable.